dali/operators/audio/preemphasis_filter_op.cc

```diff
@@ -25,7 +25,12 @@ DALI_SCHEMA(PreemphasisFilter)
 
 This filter, in simple form, can be expressed by the formula::
 
-  Y[t] = X[t] - coeff * X[t-1]
+  Y[t] = X[t] - coeff * X[t-1]        if t > 1
+
+The behavior for the best sample, depends on the ``reflect_padding`` argument::
+
+  Y[0] = X[0]                         if reflect_padding == True
```

Comment:
It's _zero_ padding, not reflect: Y[0] = X[0] - coeff * 0
Reflect would be: Y[0] = X[0] - coeff * X[1]
...and existing behavior is clamp or repeat padding.

Reply:
True, I mixed up the condition here